README.md

```diff
@@ -24,6 +24,29 @@ yarn add react-native-text-input-mask
 ```
 
 # Installation
+<details>
+  <summary><b>For RN >= 0.61</b></summary>
+
+#### iOS
+
+1. Add following lines to your target in `Podfile`.
+
+```
+use_frameworks!
+pod 'RNInputMask', :path => '../node_modules/react-native-text-input-mask/ios/InputMask'
```

Comment:
RN 0.60 added auto-linking, which means there is no need to add pods manually anymore, see https://reactnative.dev/blog/2019/07/03/version-60#native-modules-are-now-autolinked